tests/benchmark/benchmark_helpers.go

```diff
@@ -0,0 +1,661 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package benchmark
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"testing"
+
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/rpc"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+var (
+	registryProject string
+	rootResource    string
+)
+
+func init() {
+	flag.StringVar(&registryProject, "registry_project", "bench", "Name of the Registry project")
+	rootResource = fmt.Sprintf("projects/%s/locations/global", registryProject)
```

Comment:
This isn't doing what is probably intended -- because it is evaluated in the `init()` function, the `registryProject `variable still has its default value, so "rootResource" is always "projects/bench/locations/global". 

I found that inside the actual benchmarking function, e.g. `BenchmarkListApis`, `registryProject` does hold the value set by the flag, so `rootResource` could be constructed there either directly or by calling a helper function that refers to the `registryProject` variable.